experimental/devices/ina219/example_test.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package ina219_test
+
+import (
+	"fmt"
+	"log"
+
+	"periph.io/x/periph/conn/i2c/i2creg"
+	"periph.io/x/periph/conn/physic"
+	"periph.io/x/periph/experimental/devices/ina219"
+	"periph.io/x/periph/host"
+)
+
+func Example() {
+	// Make sure periph is initialized.
+	if _, err := host.Init(); err != nil {
+		log.Fatal(err)
+	}
+
+	// open default I²C bus
```

Comment:
`// Open default I²C bus.`

Please start comments with capital letter and end with a period. See for more info:
https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

Do below too.